### PR TITLE
ES-853397-Add fallback font code snippets in UG documentation

### PR DESCRIPTION
**Pull request by LokeshBaskar**
Please refer to the below MR for more details:
https://github.com/syncfusion-content/fileformat-docs/pull/1693

### CODE REVIEW

File: File-Formats/DocIO/word-to-image.md [hunk 1]
```diff
@@ -653,6 +653,7 @@ N> 5. Different styles of borders are known limitations in Word to Image convers
 N> 6. In ASP.NET Core, Blazor, Xamarin, WinUI and .NET MAUI platforms, to convert Word document to images we recommend you to use Word to image [assemblies](https://help.syncfusion.com/file-formats/docio/assemblies-required#converting-word-document-to-image) or [NuGet](https://help.syncfusion.com/file-formats/docio/nuget-packages-required#converting-word-document-to-image) as a reference in your application.
 N> 7. DocIO supports Word to image conversion in UWP application using DocIORenderer.
 N> 8. In addition to the previous NuGet packages, we recommend to use [SkiaSharp.NativeAssets.Linux v2.88.6](https://www.nuget.org/packages/SkiaSharp.NativeAssets.Linux/2.88.6) and [HarfBuzzSharp.NativeAssets.Linux v7.3.0](https://www.nuget.org/packages/HarfBuzzSharp.NativeAssets.Linux/7.3.0) NuGets to perform Word to Image conversion in Linux environment.
+N> 9. The .NET Word Library (DocIO) uses System.Drawing functionalities for Word to image conversion in .NET Framework applications. And System.Drawing itself uses fallback font to preserve the Unicode text while drawing the text in image. So, these Fallback fonts APIs are **not supported in .NET Framework**.
```
Comment (M-A32) [inline File-Formats/DocIO/word-to-image.md:656]: N> 9. The .NET Word Library (DocIO) uses System.Drawing functionalities for Word to image conversion in .NET Framework applications. And System.Drawing itself uses a fallback font to preserve the Unicode text while drawing the text in the image. So, these Fallback fonts APIs are **not supported in .NET Framework**.

File: File-Formats/DocIO/Assemblies-Required.md [hunk 2]
```diff
@@ -140,6 +140,8 @@ Syncfusion.DocIO.Portable<br/>Syncfusion.Compression.Portable<br/>Syncfusion.Off
 Syncfusion.DocIO.NET<br/>Syncfusion.Compression.NET<br/>Syncfusion.OfficeChart.NET<br/>Syncfusion.Pdf.NET<br/>Syncfusion.DocIORenderer.NET<br/>Syncfusion.Pdf.Imaging.NET<br/> SkiaSharp<br/>Syncfusion.SkiaSharpHelper.NET</td></tr>
 </table>
 
+N> Starting with v24.1.x, if you reference "Syncfusion.DocIORenderer", you also have to add "Syncfusion.Pdf.Imaging" assembly reference in your projects to perform Word to Image conversion.
```
Comment (M-A32) [inline File-Formats/DocIO/Assemblies-Required.md:143]: N> Starting with the v24.1.x, if you reference "Syncfusion.DocIORenderer", you have to add the "Syncfusion.Pdf.Imaging" assembly reference in your projects to perform Word to Image conversion.

**Review by M-A32 — CHANGES_REQUESTED** [review]
I have completed the review.

File: File-Formats/Presentation/Presentation-to-PDF.md [hunk 3]
```diff
@@ -345,7 +345,14 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Fallback fonts
 
-When a glyph of input text is unavailable in mentioned font, text will not be preserved in PPTX to PDF conversion. To avoid this, Syncfusion PowerPoint library allows you to use a fallback font to preserve the text properly in PPTX to PDF conversion.
+During PowerPoint to PDF conversions, if a glyph of the input text is unavailable in the specified font, the text will not be rendered properly. To address this, the Syncfusion PowerPoint (Presentation) library allows users to specify fallback fonts. When a glyph is missing, the library will use one of the fallback fonts to render the text correctly in the output PDF document.
+
+Users can configure fallback fonts in the following ways:
+* Initialize default fallback fonts.
+* Set custom fonts as fallback fonts for specific script types, including Arabic, Hebrew, Chinese, and Japanese etc.
```
[the remark this answers is not in the document]
Comment (LokeshBaskar) [reply File-Formats/Presentation/Presentation-to-PDF.md]: Modified the sentences

File: File-Formats/DocIO/word-to-image.md (hunk 1 above)
Comment (LokeshBaskar) [reply File-Formats/DocIO/word-to-image.md]: Modified the sentences

File: File-Formats/DocIO/Assemblies-Required.md (hunk 2 above)
Comment (LokeshBaskar) [reply File-Formats/DocIO/Assemblies-Required.md]: Modified the sentences

**Review by MohanaselvamJothi — APPROVED** [review]

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**